source/compiler/sc4.c

```diff
@@ -343,17 +343,19 @@ SC_FUNC void markexpr(optmark type,const char *name,cell offset)
  *
  *  Global references: funcstatus  (referred to only)
  */
-SC_FUNC void startfunc(char *fname)
+SC_FUNC void startfunc(char *fname, char isnaked)
```

Comment:
The styling followed by the project requires that you do not add a space between arguments.